oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/model/BanyanDBModelExtension.java

```diff
@@ -45,4 +46,23 @@ public class BanyanDBModelExtension {
     @Setter
     private boolean storeIDTag;
 
+    /**
+     * lru_size defines how many snapshots are allowed to be maintained in the memory.
+     * The default value is 2 in the BanyanDB if not set.
+     *
+     * @since 9.4.0
+     */
+    @Getter
+    @Setter
+    private int lruSize = 2;
```

Comment:
Could you explain a little more about lru_size? How does this impact?

Reply:
We should have a TopN sub-class. Building this inside `BanyanDBModelExtension` is not very clear. Especially, I think all models have BanyanDBModelExtension, but not all of them have TopN definition.

Reply:
https://github.com/apache/skywalking-banyandb/blob/ad6a81c726640c8ffb8805adac4f43ffa5b2c423/api/proto/banyandb/database/v1/schema.proto#L138.

It determines how many time_buckets are held. `2` means a data point belonging to the latest 2 time_buckets will be analyzed.

Reply:
Thanks for the explanation. I've reworded a bit for this doc